### PR TITLE
fix serenity report to filter integration , functional, Smoke Tests

### PR DESCRIPTION
**Pull request by shreedhar-hmcts**
### Change description ###
fix serenity report to filter integration , functional, Smoke Tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/DeleteUserProfileFuncTest.java [hunk 1]
```diff
@@ -21,6 +23,7 @@
 
 
 @RunWith(SpringIntegrationSerenityRunner.class)
+@WithTags({@WithTag("testType:Functional")})
```
Comment (KOTLAPRA23) [inline src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/DeleteUserProfileFuncTest.java:26]: try to add tag in the super class

**Review by KOTLAPRA23 — COMMENTED** [review]
please do the required change

File: src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/DeleteUserProfileFuncTest.java (hunk 1 above)
Comment (shreedhar-hmcts) [reply src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/DeleteUserProfileFuncTest.java:26]: already tried, it doesnt work

Comment (shreedhar-hmcts) [reply src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/DeleteUserProfileFuncTest.java:26]: following two tags needs to be together , then only tagging works, also if put it in super class then it requires at least 1 test case in that class.
@RunWith(SpringIntegrationSerenityRunner.class)
@WithTags({@WithTag("testType:Integration")})

**Review by abhijitdiwan — APPROVED** [review]

**Review by shashank-rastogi — APPROVED** [review]

**Review by pksaxena01 — APPROVED** [review]
Approved

File: src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/DeleteUserProfileFuncTest.java (hunk 1 above)
Comment (KOTLAPRA23) [reply src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/DeleteUserProfileFuncTest.java:26]: > already tried, it doesnt work

it is working tested in the PRD and please follow same in the UP

**Review by KOTLAPRA23 — APPROVED** [review]

**Review by pksaxena01 — APPROVED** [review]
Approved

**Pull request merged**